pyccel/ast/cwrapper.py

```diff
@@ -986,6 +987,24 @@ class Py_ssize_t_Cast(PythonInt):
     name = 'Py_ssize_t'
 
 
+class PyTuple_Pack(PyccelFunction):
+    """
+    A class representing a call to PyTuple_Pack.
+
+    A class representing a call to PyTuple_Pack. A class is used instead
+    of a FunctionDef as the number of arguments is variable.
```

Comment:
I suggest adding a couple of words here:
```suggestion
    A class representing a call to Python's PyTuple_Pack function.

    A class representing a call to Python's PyTuple_Pack function. A class
    is used instead of a FunctionDef as the number of arguments is variable.
```